src/clusterfuzz/_internal/cron/project_setup.py

```diff
@@ -545,7 +552,15 @@ def create_project_settings(project, info, service_account):
 
 def create_pubsub_topics(project):
   """Create pubsub topics for tasks."""
-  for platform in PUBSUB_PLATFORMS:
+  platforms = PUBSUB_PLATFORMS
+  if 'android_' not in project:
```

Comment:
Rather than add this condition, can we refactor things such that there are two functions -- one for `create_pubsub_topics_for_untrusted`, and one for `create_pubsub_topics_for_queue_id` ? That way the logic is a bit cleaner between the two cases, and if there are still significant bits of shared code we can put them into a common helper.

Reply:
Refactored into two new functions:
1. `create_pubsub_topics_for_untrusted`
2. `create_pubsub_topics_for_queue_id`

Combine the common logic into the new `_create_pubsub_topic()` helper function.